packages/clay-empty-state/src/index.tsx

```diff
@@ -93,6 +102,11 @@ const ClayEmptyState = ({
 									imgPropsReducedMotion &&
 										imgPropsReducedMotion.className
 								)}
+								onError={() => {
+									console.warn(
+										`The image url defined in \`imgSrcReducedMotion\` does not exist on the server.`
+									);
```

Comment:
We can also create a state to define whether the image was loaded or had an error, we can use the state value to return in the `useMemo` I mentioned above an empty value for `imgSrcReducedMotion` so we don't render a broken image.

For example:

```jsx
const [error, setError] = useState(false);

const fallbackImageSource = useMemo(() => {
  if (imgSrcReducedMotion) {
    return imgSrcReducedMotion;
  }

  if (error) {
    return null;
  }
}, [error, imgSrcReducedMotion]);

<img
  onError={() => setError(true)}
/>
```